modules/search/instant-search/components/search-results.scss

```diff
@@ -19,7 +20,27 @@
 	margin-right: 4em;
 }
 .jetpack-instant-search__search-results-secondary {
-	flex: 2;
+	display: none; // TODO: toggle visibility via JS?
```

Comment:
It looks like this `display: none` is never overridden in `@include break-medium()`?